test/functional/Java8andUp/src_110_up/org/openj9/test/java/lang/Test_Class.java

```diff
@@ -446,7 +446,7 @@ public void test_getMethods_subtest6() throws Exception {
 	ClzImpl clzImpl = new ClzImpl();
 	for (Method method : clzImpl.getClass().getMethods()) {
 		for (Class<?> anInterface : method.getDeclaringClass().getInterfaces()) {
-			if (IdentityInterface.class != anInterface) {
+			if (!anInterface.getName().equals("java.lang.IdentityInterface")) {
```

Comment:
This should fix the build breakage, but a more general solution is to remove the assumption that all other interfaces declare this particular method.